activesupport/lib/active_support/message_verifier.rb

```diff
@@ -24,6 +24,12 @@ module ActiveSupport
   # hash upon initialization:
   #
   #   @verifier = ActiveSupport::MessageVerifier.new('s3Krit', serializer: YAML)
+  #
+  # +MessageVerifier+ creates HMAC signatures using SHA1 hash algorithm by default.
+  # If you want to use another hash algorithm, you can change it by providing
```

Comment:
I changed it to different but I think meanings of this two words are same in this case and we can use them interchangeably.